src/zenml/cli/stack_recipes.py

```diff
@@ -1212,3 +1451,114 @@ def destroy(
                     "there's any active nodes.Ignore this warning if there "
                     "are no active nodes."
                 )
+
+
+# a function to get the value of outputs passed as input, from a stack recipe
+@stack_recipe.command(
+    name="output",
+    help="Get a specific output or a list of all outputs from a stack recipe.",
+)
+@click.argument("stack_recipe_name", type=str)
+@click.option(
+    "--path",
+    "-p",
+    type=click.STRING,
+    default="zenml_stack_recipes",
+    help="Relative path at which you want to install the stack_recipe(s)",
+)
+@click.option(
+    "--output",
+    "-o",
+    type=click.STRING,
+    default=None,
+    help="Name of the output you want to get the value of. If none is given,"
+    "all outputs are returned.",
+)
+@pass_git_stack_recipes_handler
+def get_outputs(
+    git_stack_recipes_handler: GitStackRecipesHandler,
+    stack_recipe_name: str,
+    path: str,
+    output: Optional[str],
+) -> Dict[str, str]:
+    """Get the outputs of the stack recipe at the specified relative path.
+
+    `zenml stack_recipe deploy stack_recipe_name` has to be called with the
+    same relative path before the get_outputs command.
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack_recipe.
+        path: The path of the stack recipe you want to get the outputs from.
+        output: The name of the output you want to get the value of. If none is given,
+            all outputs are returned.
+
+    Returns:
+        A dictionary with the outputs of the stack recipe.
```

Comment:
Do we want to offer different options for this output? i.e. YAML format or JSON? Maybe as an `--export` option along with the format?

Reply:
done!